### ISSUE TITLE
Season 2 Registration Deadline Reminder (Fri. Feb. 19th) 

### ISSUE THREAD

**Issue by emilmudroncik** [promazo/Content-Team#8]
@mikepromazo - Here are three different graphics I made for Friday's post. Let me know if you want anything changed.

![100k registration graphic.jpg](https://images.zenhubusercontent.com/601d6a3c366970e4db858599/6583d2db-5398-409b-b405-b1d3a34d5848)![Emily - Deadline Reminder #1.png](https://images.zenhubusercontent.com/601d6a3c366970e4db858599/240fbf1d-acab-4c6a-8355-8771cfb4b6c0)![Emily - Deadline Reminder #2.png](https://images.zenhubusercontent.com/601d6a3c366970e4db858599/f54547e1-b271-439b-a9cb-b84db465830d)

**Comment by mikepromazo** [comment]
@emilmudroncik 

We are going to share/use the graphic in the first option for mentors and mentees who were matched in Season 2 so let's use the second option.

On the second option - can you make the text smaller and not overlap with the color blob on the border?  

What text do we plan to include with the post on LI and Instagram?

I moved this Issue to Pipeline 6. Content Being Edited.

**Comment by emilmudroncik** [comment]
@mikepromazo - Here is the updated graphic and the caption I drafted up:
"Click the link in our bio to download the 100k Mentor Challenge App and register TODAY to find your perfect mentor match! Registration closes today (2/19) at 5PM CST. 

#mentor #mentee #100kMentorChallenge #ProMazo #business #mentorship #success #itsamatch"



![Revised Registration Graphic](https://user-images.githubusercontent.com/78609004/108275140-e1349800-7143-11eb-9ed1-f8d198de52ab.png)

**Comment by mikepromazo** [comment]
Let's get rid of "Download the app to get started!" below the image as the image already says download now.  Once that's updated it should be ready to post.

**Comment by emilmudroncik** [comment]
@mikepromazo - here is the updated graphic

![Emily - Deadline Reminder #1](https://user-images.githubusercontent.com/78609004/108370434-a9246800-71ca-11eb-8817-fdde472a59ee.png)

**Comment by mikepromazo** [comment]
Looks good, can you post first thing tomorrow morning?

**Comment by emilmudroncik** [comment]
Yes! Does 9am CST sound ok? And just want to make sure you want this posted on both Linkdin and Instagram, correct?

**Comment by mikepromazo** [comment]
Yup sounds great!
